docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,94 @@
+
+.Requirements
+[sidebar]
+--
+You must have _one_ of the following installed on the hosts you want to monitor:
+
+* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's status is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x or later. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
+--
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that analyzes current and emerging threats and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
+
+An indicator, also referred to as an IoC, is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-indicators-page]]
+== Set up the Indicators page
+
+Install a threat intelligence integration to add indicators to the Indicators page.
+
+
+. Choose one of the following:
+* From the {security-app} main menu, go to *Intelligence* -> *Indicators* -> *Add Integrations*.
+* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
+. Select a threat intelligence integration, then complete the integration's guided installation.
++
+NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
+
+. Return to the Indicators page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[troubleshoot-indicators-page]]
+=== Troubleshooting
+If indicator data is not appearing in the Indicators table after you installed a threat intelligence integration:
+
+* Verify that the index storing indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index storing indicator documents will differ based on the way you're collecting indicator data:
```

Comment:
Do you think it'll be more concise if we say "verify that the index that stores indicator documents are also included in the ..."? I'll leave this up to you. 

```suggestion
* Verify that the index storing indicator documents are included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index storing indicator documents will differ based on the way you're collecting indicator data:
```

Reply:
Hm, maybe not. These indices are included by default. I feel like asking users to verify that they're _also_ included could imply that they're optional indices, which they're not. For example, the `logs-ti*` index pattern doesn't only include indicator documents. It also includes documents for Logstash events and other events used by other threat intel Security features (e.g., the Threat intel card on the Overview page).